### DISCUSSION TITLE
Backlash compensation

### DISCUSSION THREAD

**Discussion by yushiromarcel** [fra589/grbl-Mega-5X#256] [Ideas]
HEllo Gauthier.
THanks a lot for your great job !

Do you think there's a simple way to merge the code of shooter64738 (https://github.com/shooter64738/grbl-mega) regarding the backlash compensation in your GRBL MEGA 5x ?

Or do you plan to do it ?

Thanks again for your great job !

**Comment by fra589** [comment]
Hi @yushiromarcel,  
  
Backlash compensation is in my TODO list since a long time, but since my CNC milling machine doesn't need backlash compensation yet, I haven't taken the time to write it yet 😃   
  
Thank you for pointing out the shooter64738's code. It pretty sound like what I envisioned. It can indeed be a good source of inspiration that will save me time on this subject. It can make me decide to quickly integrate this functionality in a soon version.  
  
@++;  
Gauthier.

**Comment by yushiromarcel** [comment]
ok thanks Gauthier.
Let me know if you digg in this way. It'd be a great pleasure for me to beta test your code :-)

Et merci encore pour ton super boulot. A+

**Comment by flori1234** [comment]
Hi,
i have some backlash-problems on my cnc. 
@yushiromarcel: have you tested the backlash-compensation of shooter64738?
Would be nice to know if it works.
Kind regards 
florian

**Reply by yushiromarcel** [reply]
Yes I did.
And it didn't work at all on my CNC. 
Can't remember exacly what's wrong but I thing an eratic toolpath...
I've not digged to understand what was wrong.
